src/lib/sse/CustomEventSource.ts

```diff
@@ -89,6 +90,14 @@ export class CustomEventSource {
     }
   }
 
+  set onclose(newOnClose) {
+    this.esOnClose = newOnClose
+    this.esOnClose = this.esOnClose.bind(this)
+    if (this.eventSource) {
+      (this.eventSource as any).onclose = this.esOnClose
+    }
```

Comment:
O think the `eventSource` doesn't have a `onclose` method
https://github.com/EventSource/eventsource/blob/61e1b19c8616aa151835a4ae599b299afb574ebf/lib/eventsource.js#L317

Reply:
sure!